ci/terraform/variables.tf

```diff
@@ -19,9 +19,17 @@ variable "vcn_cidr" {
 }
 
 variable "director_subnet_cidr" {
-    default = "10.0.1.0/24"
+    default = "10.0.7.0/24"
```

Comment:
The cidr to use  can be set in the variables file, by changing director_subnet_cidr. No edits should be needed here.

Reply:
I made the change because we are now using an existing VCN and the default subnets (for the 3 ADs) use up 10.0.0.0, 10.0.1.0 and 10.0.2.0. Will remove the defaults and force the user/pipeline to supply the values.